src/AzureRMTools.ts

```diff
@@ -1263,8 +1265,8 @@ export class AzureRMTools {
                 // Make sure the kind of item being renamed is valid
                 const result: vscode.WorkspaceEdit = new vscode.WorkspaceEdit();
                 const referenceSiteInfo: IReferenceSite | undefined = pc.getReferenceSiteInfo(true);
-                if (referenceSiteInfo && referenceSiteInfo.definition.definitionKind === DefinitionKind.BuiltinFunction) {
-                    throw new Error("Built-in functions cannot be renamed.");
+                if (referenceSiteInfo && canRename(referenceSiteInfo)) {
```

Comment:
Sorry, didn't think through that.  How about getRenameError?

Reply:
Fixed.